packages/iframe-service/src/iframe-service.ts

```diff
@@ -72,51 +101,64 @@ export const iframeService = {
                     type,
                     id,
-                    prefs: await iframeService.commands.getDefaultPreferences(),
+                    data: {
+                        prefs: await iframeService.commands.getDefaultPreferences(),
+                    },
                 });
                 break;
             case "GET_PREFERENCES":
                 iframeService.messageChild({
                     type,
                     id,
-                    prefs: await iframeService.commands.getPreferences(),
+                    data: {
+                        prefs: await iframeService.commands.getPreferences(),
+                    },
                 });
                 break;
             case "SET_PREFERENCES":
-                await iframeService.commands.setPreferences(data!.prefs);
+                await iframeService.commands.setPreferences(data.prefs);
                 iframeService.messageChild({
                     type,
                     id,
-                    prefs: await iframeService.commands.getPreferences(),
+                    data: {
+                        prefs: await iframeService.commands.getPreferences(),
+                    },
                 });
                 break;
             case "GET_TEMPLATE":
                 iframeService.messageChild({
                     type,
                     id,
-                    template: await iframeService.commands.getTemplate(),
+                    data: {
+                        template: await iframeService.commands.getTemplate(),
+                    },
                 });
                 break;
             case "GET_LOCALIZED_STRINGS":
                 iframeService.messageChild({
                     type,
                     id,
-                    strings: await iframeService.commands.getLocalizedStrings(),
+                    data: {
+                        strings:
+                            await iframeService.commands.getLocalizedStrings(),
+                    },
                 });
                 break;
             case "SEND_EMAILS":
-                await iframeService.commands.sendEmails(data!.emails);
+                if (data.emails)
+                    await iframeService.commands.sendEmails(data.emails);
                 iframeService.messageChild({ type, id });
                 break;
             case "SEND_EMAIL":
-                await iframeService.commands.sendEmail(
-                    data!.email,
-                    data!.sendmode
-                );
+                if (data.email)
+                    await iframeService.commands.sendEmail(
+                        data.email,
+                        data.sendmode
+                    );
                 iframeService.messageChild({ type, id });
                 break;
             case "OPEN_URL":
-                iframeService.commands.openUrl(data!.url);
+                if (data.url) iframeService.commands.openUrl(data.url);
```

Comment:
Could you please use curly braces around the body of `if` statements?

Reply:
Ah, sorry about that, think I got them all corrected in the last commit.